metatensor-torch/include/metatensor/torch/atomistic/model.hpp

```diff
@@ -49,10 +49,10 @@ class METATENSOR_TORCH_EXPORT ModelOutputHolder: public torch::CustomClassHolder
     ModelOutputHolder(
         std::string quantity,
         std::string unit,
-        bool per_atom_,
+        std::vector<std::string> sample_kind_,
```

Comment:
I would take a `torch::IValue` here, and allow it to be either a single string or a list of string. And then make the corresponding getter return a single string if the list contains a single element. This should leave the basic case simple while enabling more complex use cases.